StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewController.swift

```diff
@@ -0,0 +1,378 @@
+        }
+        else {
+            // if expiry date shows an error, don't show the label
+            if viewModel.paymentMethod.type == .card {
+                switch cardSection.validationState {
+                case .valid:
+                    label.isHidden = false
+                default:
+                    label.isHidden = true
+                }
+            }
+            label.isHidden = false
+        }
+        return label
+    }()
+
+    private lazy var errorLabel: UILabel = {
+        let label = ElementsUI.makeErrorLabel(theme: appearance.asElementsTheme)
+        label.isHidden = true
+        return label
+    }()
+
+    private lazy var cardBrandDropDown: DropdownFieldElement? = {
+        if viewModel.paymentMethod.isCoBrandedCard {
+            let cardBrands = viewModel.paymentMethod.card?.networks?.available.map({ STPCard.brand(from: $0) }).filter { cardBrandFilter.isAccepted(cardBrand: $0) } ?? []
+            let cardBrandDropDown = DropdownFieldElement.makeCardBrandDropdown(cardBrands: Set<STPCardBrand>(cardBrands),
+                                                                               theme: appearance.asElementsTheme,
+                                                                               includePlaceholder: false) { [weak self] in
+                guard let self = self else { return }
+                let selectedCardBrand = self.cardBrandDropDown?.selectedItem.rawData.toCardBrand ?? .unknown
+                let params = ["selected_card_brand": STPCardBrandUtilities.apiValue(from: selectedCardBrand), "cbc_event_source": "edit"]
+                STPAnalyticsClient.sharedClient.logPaymentSheetEvent(event: self.hostedSurface.analyticEvent(for: .openCardBrandDropdown),
+                                                                     params: params)
+            } didTapClose: { [weak self] in
+                guard let self = self else { return }
+                let selectedCardBrand = self.cardBrandDropDown?.selectedItem.rawData.toCardBrand ?? .unknown
+                STPAnalyticsClient.sharedClient.logPaymentSheetEvent(event: self.hostedSurface.analyticEvent(for: .closeCardBrandDropDown),
+                                                                     params: ["selected_card_brand": STPCardBrandUtilities.apiValue(from: selectedCardBrand)])
+            }
+            
+            // pre-select current card brand
+            if let currentCardBrand = viewModel.paymentMethod.card?.preferredDisplayBrand,
+               let indexToSelect = cardBrandDropDown.items.firstIndex(where: { $0.rawData == STPCardBrandUtilities.apiValue(from: currentCardBrand) }) {
+                cardBrandDropDown.select(index: indexToSelect, shouldAutoAdvance: false)
+            }
+            
+            return cardBrandDropDown
+        }
+        return nil
+    }()
+
+    private lazy var cardSection: SectionElement = {
+        lazy var panElement: TextFieldElement = {
```

Comment:
Same comment about any lazy's inside of a lazy, I haven't seen that before and don't think we need it.